File: fsharp-functions/OpenAPI/src/model/RealmDeactivatedError.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.AnyType
open OpenAPI.Model.CodedError

module RealmDeactivatedError = 

  //#region RealmDeactivatedError

  [<CLIMutable>]
  type RealmDeactivatedError = {
    [<JsonProperty(PropertyName = "result")>]
    Result : AnyType;
    [<JsonProperty(PropertyName = "msg")>]
    Msg : AnyType;
    [<JsonProperty(PropertyName = "code")>]
    Code : AnyType;
  }
  
  //#endregion
  

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/RealmDeactivatedError.scala
package org.openapitools.server.model


/**
 * @param result  for example: ''null''
 * @param msg  for example: ''null''
 * @param code  for example: ''null''
*/
final case class RealmDeactivatedError (
  result: AnyType,
  msg: AnyType,
  code: Option[AnyType]
)

